assembly/JSON.ts

```diff
@@ -183,24 +183,30 @@ export abstract class Value {
 }
 
 export class Str extends Value {
+  private escape_map:Map<string, string> = new Map<string, string>();
+
   constructor(public _str: string) {
     super();
+    const from: Array<string> = ['"', "\\", "\b", "\n", "\r", "\t", '\f', '\v'];
+    const to: Array<string> = ['\\"', "\\\\", "\\b", "\\n", "\\r", "\\t", "\\f", "\\u000b"];
+    
+    for(let i=0; i < from.length;i++){
+      this.escape_map.set(from[i], to[i]);
+    }
   }
 
   stringify(): string {
-    let escaped: i32[] = [];
```

Comment:
We also need to check if this original strategy was needed for safety.  It seems like a good change, but I think perhaps @MaxGraey would know?

Reply:
@MaxGraey Is there a reason for using code points vs chars?

Reply:
If it's arbitrary text for example fields or content better use code points. But it it's escaped char like `\n`, `\\` or `{` better use `charCodeAt` which is slightly faster

Reply:
ok, will change it back to i32[]

Reply:
Working with separate code points always faster than array of chars or strings

Reply:
I've just noticed that the encoder also uses an Array<string> while encoding/escaping strings. I would suggest that we leave this as a string, and in a further change, we modify both cases (the function using `this.result` from encoder.ts and this `stringify()` function.